recipe/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
Hmm...so a bunch of files appear to have been added like this one. 😕 Guessing this came from a bad re-rendering. Can take a look locally and push a fix if you would like.